services/ui-src/src/components/modals/AddEditReportModal.tsx

```diff
@@ -76,9 +80,35 @@ export const AddEditReportModal = ({
     setForm(customizedModalForm);
   }, [selectedReport, copyEligibleReportsByState]);
 
+  // handle program name selection from MCPAR Program List
+  const onChange = (event: InputChangeEvent) => {
+    if (reportType === ReportType.MCPAR) {
+      // make deep copy of baseline form for customization
+      let customizedModalForm: FormJson = JSON.parse(
+        JSON.stringify(modalFormJson)
```

Comment:
This seems redundant? Isn't it doing object -> string -> object? If it's not acting correctly, i think you might just need to cast it.

Reply:
true, I copy-pasted from above and didn't catch that. thanks!